frontend/test/__support__/ui.tsx

```diff
@@ -12,29 +12,35 @@ import { Router, useRouterHistory } from "react-router";
 import { routerReducer, routerMiddleware } from "react-router-redux";
 import _ from "underscore";
 
+import { AppInitializeController } from "embedding-sdk/components/private/AppInitializeController";
+import { SDK_REDUCERS } from "embedding-sdk/store";
+import type { SDKConfigType } from "embedding-sdk/types";
 import { Api } from "metabase/api";
 import { UndoListing } from "metabase/containers/UndoListing";
 import mainReducers from "metabase/reducers-main";
 import publicReducers from "metabase/reducers-public";
+import { EmotionCacheProvider } from "metabase/styled-components/components/EmotionCacheProvider";
 import { ThemeProvider } from "metabase/ui";
 import type { State } from "metabase-types/store";
 import { createMockState } from "metabase-types/store/mocks";
 
 import { getStore } from "./entities-store";
 
 type ReducerValue = ReducerObject | Reducer;
+
 interface ReducerObject {
   [slice: string]: ReducerValue;
 }
 
 export interface RenderWithProvidersOptions {
-  mode?: "default" | "public";
+  mode?: "default" | "public" | "sdk";
```

Comment:
nit: a doc comment here might be nice to understand what the mode configuration here does - a quick look suggests me that this changes the set of reducers and providers applied - might be nice to have that written down.

Reply:
added a quick comment there